eng/versioning/version_client.txt

```diff
@@ -68,6 +68,11 @@ com.microsoft.azure:spring-cloud-azure-appconfiguration-config-web;1.2.8-beta.1;
 com.microsoft.azure:spring-cloud-azure-appconfiguration-config;1.2.8-beta.1;1.2.8-beta.1
 com.microsoft.azure:spring-cloud-azure-context;1.2.8-beta.1;1.2.8-beta.1
 com.microsoft.azure:spring-cloud-azure-telemetry;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-integration-azure-core;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-integration-azure-test;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-integration-eventhubs;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-integration-servicebus;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-integration-storage-queue;1.2.8-beta.1;1.2.8-beta.1
```

Comment:
@jialindai at this point should these groupdIds be com.azure with artifactIds following the naming convention?

Reply:
@JimSuplizio I've discussed with @jialindai and the groupIds / artifactIds will not be changed in this PR. And since these spring-integration-xxx modules are part of the Spring Cloud project, I'll update [this issue](https://github.com/Azure/azure-sdk-for-java/issues/13523).